src/main/resources/marc4bibframe.yml

```diff
@@ -1760,3 +1672,20 @@ bibSharedRules:
           LINK: http://id.loc.gov/vocabulary/organizations/dlc
         marc2ldCondition:
           ind2: '0'
+  26X_mappings:
+    types: PROVIDER_EVENT
+    parent: INSTANCE
+    subfields:
+      b: NAME
+    controlFields:
+      '008':
+        PROVIDER_DATE: 7,11
+    edges:
+      - types: PLACE
+        parent: PROVIDER_EVENT
+        predicate: PROVIDER_PLACE
+        controlFields:
+          '008':
+            CODE: 15,18
+        constants:
+          LINK: http://id.loc.gov/vocabulary/countries/%CODE
```

Comment:
shouldn't we add here?
```
ld2marcCondition:
    skip: true
```

Reply:
`ld2marcCondition -> skip: true` will only skip the "edges" when deriving MARC. It do not have any effect on "controlFields". As we do not have any "edges" at this level, `ld2marcCondition -> skip: true` will not have any effect here.